bucket/prismlauncher.json

```diff
@@ -1,14 +1,21 @@
 {
-    "version": "5.2",
-    "description": "An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.",
+    "version": "6.0",
+    "description": "An open sourced Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "architecture": {
```

Comment:
It was set as 64bit only on purpose, plz fix

Reply:
what about on the legacy version?